criu/flog.c

```diff
@@ -200,11 +224,17 @@ int flog_encode_msg(int fdout, unsigned int nargs, unsigned int mask, const char
 
 	m->size = roundup(m->size, 8);
 	if (mbuf == _mbuf) {
-		if (flog_enqueue(m))
+		if (flog_enqueue(fdout, m))
 			return -1;
 	} else {
 		mbuf += m->size;
 		mbuf_size -= m->size;
 	}
 	return 0;
-}
\ No newline at end of file
+
+	regular_logging:
```

Comment:
Since this is a goto-only code, then it's worth just moving into the respective if () branch. I thought you re-used some parts of this function in both if and else branches in the previous version of this patch, that's why asked for goto that time.

Reply:
Alright Sir, I will make this change.
The if statement here determines whether the binary logging option is enabled. If it is, then we use the flog printer to store the messages into the file. If it is not, then the criu/log.c file’s vprint_on_level is called to log via the regular printer.